packages/io/svg-deserializer/index.js

```diff
@@ -38,7 +38,8 @@ const deserialize = (options, input) => {
     output: 'script',
     pxPmm: require('./constants').pxPmm,
     target: 'path', // target - 'geom2' or 'path'
-    version: '0.0.0'
+    version: '0.0.0',
+    segments: 16
```

Comment:
Let's use segments 32, as that's the value for all the normal primitives, etc.

Note... this will require changes to the test cases.